openlabcmd/openlabcmd/zk.py

```diff
@@ -19,12 +19,14 @@
     'dns_provider_api_url': 'https://api.dnsimple.com/v2/',
     'dns_provider_token': None,
     'dns_status_domain': 'test-status.openlabtesting.org',
+    'dns_ori_web_ip': None,
+    'dns_change_web_ip': None,
```

Comment:
let's use `dns_master_public_ip` and `dns_slave_public_ip`

Reply:
Done